library/alloc/src/vec/mod.rs

```diff
@@ -2274,6 +2274,23 @@ impl<T: Clone, A: Allocator> Vec<T, A> {
     }
 }
 
+impl<T, A: Allocator, const N: usize> Vec<[T; N], A> {
+    #[unstable(feature = "slice_flatten", issue = "none")]
+    pub fn into_flattened(self) -> Vec<T, A> {
+        let (ptr, len, cap, alloc) = self.into_raw_parts_with_alloc();
+        let new_len = if crate::mem::size_of::<T>() == 0 {
```

Comment:
This is failing because this is `alloc`, not `core` or `std`, so `mem` isn't in this crate.

But the module is `use`d already in this file, so you want
```suggestion
        let new_len = if mem::size_of::<T>() == 0 {
```